docs/sphinx/examples/python/tutorials/Shors.ipynb

```diff
@@ -414,35 +425,35 @@
    "id": "0c644786",
    "metadata": {},
    "source": [
-    "Verify in the code block below that the kernels defined above do in fact carry out multiplication and exponentiation with $a=5$ and $N=21$. "
+    "Verify in the code block below that the kernels defined above do in fact carry out multiplication and exponentiation with $a=5$ and $N=21$ by changing the `iterations` variable below. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 93,
    "id": "e93fcf39",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "     ╭───╮╭───╮   ╭───╮      ╭───╮      ╭───╮      ╭───╮      \n",
-      "q0 : ┤ x ├┤ x ├─╳─┤ x ├────╳─┤ x ├────╳─┤ x ├────╳─┤ x ├────╳─\n",
-      "     ╰───╯╰───╯ │ ╰───╯    │ ╰───╯    │ ╰───╯    │ ╰───╯    │ \n",
-      "q1 : ───────────┼──────────┼──────────┼──────────┼──────────┼─\n",
-      "     ╭───╮      │ ╭───╮    │ ╭───╮    │ ╭───╮    │ ╭───╮    │ \n",
-      "q2 : ┤ x ├──╳───┼─┤ x ├─╳──┼─┤ x ├─╳──┼─┤ x ├─╳──┼─┤ x ├─╳──┼─\n",
-      "     ╰───╯  │   │ ╰───╯ │  │ ╰───╯ │  │ ╰───╯ │  │ ╰───╯ │  │ \n",
-      "q3 : ───────┼───┼───────┼──┼───────┼──┼───────┼──┼───────┼──┼─\n",
-      "     ╭───╮  │   │ ╭───╮ │  │ ╭───╮ │  │ ╭───╮ │  │ ╭───╮ │  │ \n",
-      "q4 : ┤ x ├──╳───╳─┤ x ├─╳──╳─┤ x ├─╳──╳─┤ x ├─╳──╳─┤ x ├─╳──╳─\n",
-      "     ╰───╯        ╰───╯      ╰───╯      ╰───╯      ╰───╯      \n",
+      "     ╭───╮╭───╮        ╭───╮        ╭───╮      \n",
+      "q0 : ┤ x ├┤ x ├─╳───╳──┤ x ├─╳───╳──┤ x ├─╳──╳─\n",
+      "     ╰───╯╰───╯ │   │  ╰───╯ │   │  ╰───╯ │  │ \n",
+      "q1 : ───────────┼───┼────────┼───┼────────┼──┼─\n",
+      "     ╭───╮      │   │  ╭───╮ │   │  ╭───╮ │  │ \n",
+      "q2 : ┤ x ├──────┼───╳──┤ x ├─┼───╳──┤ x ├─┼──╳─\n",
+      "     ╰───╯      │      ╰───╯ │      ╰───╯ │    \n",
+      "q3 : ───────────┼────────────┼────────────┼────\n",
+      "     ╭───╮      │ ╭───╮      │ ╭───╮      │    \n",
+      "q4 : ┤ x ├──────╳─┤ x ├──────╳─┤ x ├──────╳────\n",
+      "     ╰───╯        ╰───╯        ╰───╯           \n",
       "\n",
-      "Measurement results: { 10001:200 }\n",
+      "Measurement results in big-endian notation: { 00101:200 }\n",
       "\n",
-      "For x =5, 5^x mod 21 = 17\n",
-      "For x =5,the computed result of the circuit is 17\n"
+      "For x = 3, 5^x mod 21 = 20\n",
+      "For x = 3, the computed result of the circuit is 20\n"
```

Comment:
Just to elaborate, I agree that in CUDA-Q, one could say that `00101` corresponds to `20`, because we print the bitstrings left to right, lowest index to highest index. So, if the bitstring indices $i$ correspond to the $2^i$ position values,  `00101` corresponds to $0 \times 2^0 + 0 \times 2^1 + 1 \times 2^2 + 0 \times 2^3 + 1 \times 2^4 = 20$, I would just call that a little endian notation because we are printing the LSB first, which is what you would expect in a little endian convention.

Reply:
OK.  Maybe I'll just remove the "endian" terminology altogether and just be more explicit about how things are printed out and stored in qubits